bio.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Kevin's Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+  </head>
+  <body>
+    <header>
+        <nav>
+            <a class="nav-links" href="index.html"><h1>Kevin's Portfolio</h1></a>
+            <div class="links">
+            <a class="nav-links" href="bio.html"><h3>Biography</h3></a>
+            <a class="nav-links" href="portfolio.html"><h3>Portfolio</h3></a>
+            </div> 
+        </nav>
+    </header>
+    <main>
```

Comment:
Effective use of semantic html - you are using the different elements correctly and structuring your html clearly inline with best practices.  One small misconception: header tag should only appear once on a page, which indicates the header at the top of the page.  Additional headings are marked with h2,h3, h4 etc.